src/components/SlippageWarningNote/index.tsx

```diff
@@ -31,24 +31,15 @@ const TextUnderlineTransparent = styled(Text)`
   display: inline;
 `
 
-const SlippageWarningNote: FC<Props> = ({ className, rawSlippage, isStablePairSwap, isCorrelatedPair }) => {
-  const slippageStatus = checkRangeSlippage(rawSlippage, isStablePairSwap, isCorrelatedPair)
+const SlippageWarningNote: FC<Props> = ({ className, rawSlippage }) => {
+  const cat = usePairCategory()
+  const slippageStatus = checkRangeSlippage(rawSlippage, cat)
 
   if (slippageStatus === SLIPPAGE_STATUS.NORMAL) {
     return null
   }
 
-  let msg = 'setting might be high. You might want to adjust it to avoid potential front-running.'
-  if (slippageStatus === SLIPPAGE_STATUS.LOW) {
-    msg = 'is low. Your transaction may fail.'
-  }
-
-  if (isStablePairSwap)
-    msg =
-      'setting might be high compared to typical stable pair trades. Consider adjusting it to reduce the risk of front-running.'
-  if (isCorrelatedPair)
-    msg =
-      'setting might be high compared with other similar trades. You might want to adjust it to avoid potential front-running.'
+  const msg = (SLIPPAGE_WARNING_MESSAGES[slippageStatus] as Record<PAIR_CATEGORY, string>)[cat] || ''
 
```

Comment:
@viet-nv 
1. I checked the 2024 slippage spec and it said that there is no warning for lower slippage for the exotic pair, please check again
2. This if-else logic could be improved and I have pushed a commit to do it